### PR TITLE
runtime(doc): Changes in documentation

### PR DESCRIPTION
**Pull request by RestorerZ**
A few adjustments to the documentation

### CODE REVIEW

File: runtime/doc/intro.txt [hunk 1]
```diff
@@ -133,7 +133,7 @@ There are three ways to report bugs:
    This is a maillist, you need to become a member first and many people will
    see the message.  If you don't want that, e.g. because it is a security
    issue, please contact any of the current Vim maintainers
-   https://github.com/orgs/vim/people (but not Bram or the vim-dev ML).
+   https://github.com/orgs/vim/people (but not the vim-dev ML).
```
Comment (k-takata) [inline runtime/doc/intro.txt]: Bram's account and the vim-dev ML account used to be listed in this URL.
However, they are now private and not visible for non-vim-org members.
So, the comment in the parentheses is not needed anymore?

File: runtime/doc/helphelp.txt [hunk 2]
```diff
@@ -315,7 +315,7 @@ Hints for translators:
   using the "tag@en" notation.
 - Make a package with all the files and the tags file available for download.
   Users can drop it in one of the "doc" directories and start use it.
-  Report this to Bram, so that he can add a link on www.vim.org.
+  Report to the development team, so they can add a link on www.vim.org.
```
Comment (Shane-XB-Qian) [inline runtime/doc/helphelp.txt:318]: it seems called `the vim project team`.
// and tho the info is out-of-date, but somewhere kept the information or markers or name of Bram it ok.
// e.g `the vim project team, successor of Bram.`

Comment (Shane-XB-Qian) [inline runtime/doc/helphelp.txt:318]: it seems called `the vim project team`.
// and tho the info is out-of-date, but somewhere kept the information or markers or name of Bram is ok.
// e.g `the vim project team, successor of Bram.`

File: runtime/doc/intro.txt (hunk 1 above)
Comment (RestorerZ) [reply runtime/doc/intro.txt]: Then perhaps it's better to change it at all as stated in SECURITY.md? It's all about security messages.

File: runtime/doc/helphelp.txt (hunk 2 above)
Comment (RestorerZ) [reply runtime/doc/helphelp.txt:318]: > it seems called `the vim project team`. // and tho the info is out-of-date, but somewhere kept the information or markers or name of Bram is ok. // e.g `the vim project team, successor of Bram.`

Yes, it's probably better. But then I guess "The Vim Project" then, as it is specified in PO files, for example.

Comment (Shane-XB-Qian) [reply runtime/doc/helphelp.txt:318]: then better uniform it, :smile:

**Comment by chrisbra** [comment]
Thanks. I have made a few more changes, since the voting feature is no longer possible.

**Comment by RestorerZ** [comment]
That's so much better! Thank you!

**Pull request closed without merging**